src/core/geocms/geonode/qgsgeonoderequest.h

```diff
@@ -213,6 +217,11 @@ class CORE_EXPORT QgsGeoNodeRequest : public QObject
      */
     void setProtocol( const QString &protocol );
 
+    /**
+     * Returns the updated ServiceLayerDetail struct with WMS/WFS/XYZ url.
+     */
+    QgsGeoNodeRequest::ServiceLayerDetail parseOWSUrl( QgsGeoNodeRequest::ServiceLayerDetail &layerStruct, const QVariantList &layerLinks );
```

Comment:
I think the QGIS convention is to capitalize acronyms
```suggestion
    QgsGeoNodeRequest::ServiceLayerDetail parseOwsUrl( QgsGeoNodeRequest::ServiceLayerDetail &layerStruct, const QVariantList &layerLinks );
```